### PR TITLE
[SPARK-15935][PySpark]Fix a wrong format tag in the error message

### PR DESCRIPTION
**Pull request by zsxwing**
## What changes were proposed in this pull request?

A follow up PR for #13655 to fix a wrong format tag.
## How was this patch tested?

Jenkins unit tests.

### CODE REVIEW

**Comment by brkyvz** [comment]
LGTM! Thanks

File: python/pyspark/sql/tests.py [hunk 1]
```diff
@@ -1711,7 +1711,4 @@ def test_collect_functions(self):
 
 if __name__ == "__main__":
     from pyspark.sql.tests import *
-    if xmlrunner:
```
Comment (srowen) [inline python/pyspark/sql/tests.py]: Was this change intended?

Comment (zsxwing) [inline python/pyspark/sql/tests.py]: I'm debugging the failure. Cannot reproduce it locally. There is a xml runner issue (https://github.com/xmlrunner/unittest-xml-reporting/issues/83) that swallows the real error, so I just removed it to debug the failure.

File: dev/run-tests.py [hunk 2]
```diff
@@ -335,7 +335,7 @@ def build_spark_maven(hadoop_version):
 def build_spark_sbt(hadoop_version):
     # Enable all of the profiles for the build:
     build_profiles = get_hadoop_profiles(hadoop_version) + modules.root.build_profile_flags
-    sbt_goals = ["package",
+    sbt_goals = ["test:package",  # Build test jars as some tests depend on them
```
Comment (zsxwing) [inline dev/run-tests.py]: /cc @JoshRosen for this change

Comment (JoshRosen) [inline dev/run-tests.py]: I think that this is correct. LGTM pending tests.

**Comment by zsxwing** [comment]
Thanks. Merging into master and 2.0.
